tests/Liuggio/StatsdClient/KeymetricTest.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace Liuggio\StatsdClient;
+
+use Liuggio\StatsdClient\Factory\StatsdDataFactory,
+    Liuggio\StatsdClient\Service\StatsdService;
+
+class KeymetricTest extends \PHPUnit_Framework_TestCase
```

Comment:
The correct location for this test would be within tests/Liuggio/StatsdClient/Service/StatsdServiceTest.php (because you altered the StatsdServiceTest.php behavior).